### ISSUE TITLE
Lock does not work in IE WebView

### ISSUE THREAD

**Issue by gdestuynder** [mozilla-iam/auth0-deploy#153]
Such as GDrive Windows app login which uses the IE WebView
Upgrading the lock lib in theory fixes it due to the fix in https://github.com/auth0/lock/pull/1039 for example

**Comment by gdestuynder** [comment]
![image](https://user-images.githubusercontent.com/956866/31154358-722c7176-a85b-11e7-9b44-a6d43e6e537a.png)
Example of the issue

**Comment by hmitsch** [comment]
@gdestuynder: Do we consider this a critical issue that needs immediate addressing? If I remember correctly, the last time we intended to upgrade the lock a lot of other things broke. So this would be a non-trivial solution, right?

**Comment by viorelaioia** [comment]
Hey, @gdestuynder ! Can I get more information on how to reproduce the issue (as in what device)? 
From what I understood from the description above, I downloaded the google drive application on a Windows 10 machine and I managed to login successfully, using my ldap account. 
Thanks!

**Comment by gene1wood** [comment]
@hmitsch To do the lock upgrade to resolve this issue we would first need to solve #141 which is currently closed because the PR was reverted.

**Comment by gdestuynder** [comment]
@hmitsch correct, upgrade means quite a bit of work as @gene1wood mentionned - id rather see that effort go into the lock rewrite. Not sure if its considered critical, will loop in some people here to ask

@viorelaioia you need Windows 10 and to install the GDrive business  app: https://www.google.com/drive/download/ then just try to login with an @mozilla.com email

**Comment by mozjen** [comment]
I would say this is a Mid-High urgency.  This is new feature by Google that came out last week.  And there's approx 250 users who have non OSX machines that would be affected by this bug.   But as this feature gets adopted more widely that urgency will go up.

**Comment by gdestuynder** [comment]
@hmitsch as per @mozjen we should make sure this is tested/functional in new lock then.
Note that the Gsync issue with duo should also be tested in new lock (the thing that prevents one to use google sync in chrome) as it's a relatively easy fix (it has to call duo directly instead of sending the user to do, so that it doesn't trip chrome's sync login restrictions)

**Comment by hmitsch** [comment]
@gdestuynder thanks for the update. Makes sense. I hope that we have talent on board for the Lock Refresh very soon.

@viorelaioia were you able to reproduce the issue?

Best regards,
   Henrik

**Comment by viorelaioia** [comment]
@hmitsch , sorry, it took me some time to get access to a windows 10 machine.
I have been able to reproduce the issue by following the next steps:
1. Download and install [GDrive File Stream](https://dl.google.com/drive-file-stream/GoogleDriveFSSetup.exe) on a Windows 10 machine
2. Launch GDrive FS
3. Add mozilla email and password
4. Click Log in button

Expected results:
Duo window is shown

Actual results:
The content of the password field is removed, "Can't be blank" error message is shown. See [video](https://streamable.com/1jcww)

Note:
I tried same steps on a Mac machine and I didn't encounter this issue.

**Comment by andrewkrug** [comment]
Do we have any data about how many of our users make use of the GDrive File Stream client?  I know we have a number of users with Windows machines.  But it may be an even smaller user base that use the GDrive client.
